Makefile

```diff
@@ -1,30 +1,12 @@
-### Release configuration #################################
-# Path to folder in S3 (without slash at end)
-BUCKET = s3://dist.dividat.ch/releases/driver2
-
-# where the BUCKET folder is accessible for getting updates (needs to end with a slash)
-RELEASE_URL = https://dist.dividat.com/releases/driver2/
-
-
 ### Basic setup ###########################################
-# Set GOPATH to repository path
-CWD = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
-GOPATH ?= $(CWD)
-
 # Main source to build
 SRC = ./src/dividat-driver/main.go
 
-# Default location where built binary will be placed
-OUT ?= bin/dividat-driver
-
-# Get version from git
-VERSION := $(shell git describe --always HEAD)
+# Get version from git if not already set
+VERSION ?= $(shell git describe --always HEAD)
```

Comment:
Now `bin/dividat-driver` has different version from build artifacts.

Reply:
No, the `bin/dividat-driver` that is built from a development shell has a different version. `bin/dividat-driver` in a build has the same version as release artifacts.